src/channel.ts

```diff
@@ -66,32 +66,36 @@ export const HREF: 'href' = 'href';
 
 export const URL: 'url' = 'url';
 
-export type PositionChannel = 'x' | 'y' | 'x2' | 'y2';
-
-const POSITION_CHANNEL_INDEX: Flag<PositionChannel> = {
+const POSITION_CHANNEL_INDEX = {
```

Comment:
FWIW, you declared 

```
const PositionScaleChannel = {
  x: 1,
  y: 1
} as const;
```

below, so perhaps we should be consistent.

Reply:
Let me fix the one below. I went back and forth between the two styles.